src/v2/Components/NavBar/NavBar.tsx

```diff
@@ -45,6 +45,22 @@ import Events from "v2/Utils/Events"
 import { useMatchMedia } from "v2/Utils/Hooks/useMatchMedia"
 import { userHasLabFeature } from "v2/Utils/user"
 
+// TODO: Need a place to put global components like this
+const GlobalStyles = createGlobalStyle`
+  /**
+  * This will hide the focus indicator if the element receives focus via the mouse,
+  * but it will still show up on keyboard focus. <input> tags are an exception to this.
+  */
+  .js-focus-visible :focus:not(.focus-visible) {
+    outline: 0;
+  }
+
+  /* Custom outlines on :focus-visible */
+  .focus-visible {
+    outline: 1px solid ${color("purple100")};
+  }
+`
+
```

Comment:
@damassi I figured adding it here was better than the main .styl index. Is there a more appropriate global component? I could also split across the two `AppShell` / `StitchWrapper`.

Reply:
https://github.com/artsy/force/blob/master/src/v2/Artsy/Router/Boot.tsx#L31

Reply:
Yeah — I did that first but yeah we need someplace that's in both app contexts, hence the nav. But yeah, just copy/paste into the two places is also fine with me.

Reply:
Agreed this spot seems off if it applies to the entire page shell.

Reply:
OK, I extracted and just inserted it into the two more appropriate global locations. Take a look. @damassi depends on https://github.com/artsy/palette/pull/724 — let me know if there's a more appropriate location.